zerver/tests/test_events.py

```diff
@@ -1778,6 +1781,27 @@ def test_has_zoom_token(self) -> None:
         events = self.verify_action(lambda: do_set_zoom_token(self.user_profile, None))
         check_has_zoom_token("events[0]", events[0], value=False)
 
+    def test_stream_user_group_access_events(self) -> None:
+        hamlet = self.example_user("hamlet")
+        stream = self.make_stream("test_stream")
+        do_change_stream_post_policy(stream, Stream.STREAM_POST_POLICY_ADMINS_AND_USERGROUPS)
+        check_add_user_group(self.user_profile.realm, "test_group", [hamlet], "testing group")
+        group = UserGroup.objects.get(name="test_group")
+
+        # test create event:
+        action = lambda: update_stream_user_group_access_for_post(
+            stream, add=[group.id], remove=[], user_profile=self.user_profile
+        )
+        events = self.verify_action(action, num_events=1)
+        check_stream_user_group_access_create("events[0]", events[0])
+
+        # test delete event:
+        action = lambda: update_stream_user_group_access_for_post(
+            stream, add=[], remove=[group.id], user_profile=self.user_profile
+        )
+        events = self.verify_action(action, num_events=1)
```

Comment:
I think it should be fine to not specify `num_events` here. (It will default to one in `verify_action`)